### PR TITLE
Add bosonai/Qwen3-ASR-1.7B-hf-orze

### PR DESCRIPTION
**Pull request by warlockee**
## Description

Adds `bosonai/Qwen3-ASR-1.7B-hf-orze` to the existing native Transformers Qwen3-ASR HF Jobs submission script.

Model: https://huggingface.co/bosonai/Qwen3-ASR-1.7B-hf-orze

Exact-scored model revision: https://huggingface.co/bosonai/Qwen3-ASR-1.7B-hf-orze/commit/bce834cee3d106a9f0ec1ba98e96d995f13f59de

The checkpoint is a self-contained merged English specialization of `Qwen/Qwen3-ASR-1.7B-hf`. It requires no custom code and uses the existing `hf-audio/open-asr-leaderboard-transformers` evaluator. Its generation config uses deterministic beam size 4 and length penalty 0.7.

Exact A100 results with the same decoding settings on every dataset, rescored from the complete manifests with the leaderboard normalizer at `d1e99b25524814332d6868a5645e568670834cfb`:

| Dataset | WER |
|---|---:|
| AMI Cleaned | 7.98 |
| Earnings22 Cleaned AA chunked | 5.78 |
| GigaSpeech Cleaned | 7.18 |
| LibriSpeech test-clean | 1.19 |
| LibriSpeech test-other | 2.75 |
| SPGISpeech | 2.70 |
| VoxPopuli Cleaned AA | 2.73 |
| **Mean** | **4.33** |

RTFx: **74.01** on A100-80GB.

Compared with length penalty 0.8 on the same weights, length penalty 0.7 improved three independent, evaluation-only AppTek multi-accent call selections from 11.368 / 7.024 / 9.216 to 11.274 / 7.024 / 9.216 macro WER, and Switchboard test (first 1,000) from 9.18 to 9.15. No AppTek audio or private leaderboard data was used for training.

## Type of change

- [x] New model
- [ ] New dataset
- [ ] Bug fix
- [ ] New feature
- [ ] Other

## New Model Checklist

- [x] Results are reported in [`.eval_results/open_asr_leaderboard.yaml`](https://huggingface.co/bosonai/Qwen3-ASR-1.7B-hf-orze/blob/main/.eval_results/open_asr_leaderboard.yaml).
- [x] Uses the existing Transformers evaluation Space and batched `run_eval.py`.
- [x] Uses identical decoding hyperparameters across datasets (deterministic beam size 4, length penalty 0.7).
- [x] Exact public evaluation completed on A100-80GB with batch size 16.
- [x] Public WERs and RTFx are reported above and on the model card.
- [x] Total model size is approximately 2.04B parameters.
- [x] Training data are disclosed on the [model card](https://huggingface.co/bosonai/Qwen3-ASR-1.7B-hf-orze#training-disclosure).
- [x] Apache-2.0 license appears on the model card.

| License | Size (B) | # Languages | Encoder | Decoder | Training disclosure |
|---|---:|---:|---|---|---|
| apache-2.0 | 2.04 | 52 (English specialization evaluated) | Custom | Qwen3 | [model card](https://huggingface.co/bosonai/Qwen3-ASR-1.7B-hf-orze#training-disclosure) |

### CODE REVIEW

**Comment by warlockee** [comment]
Submission artifacts are ready for the private-suite run. The model is pinned at bosonai/Qwen3-ASR-1.7B-hf-orze revision 9e0e280741315a6b5c550858c8380006e9166c2f; exact official public evaluation at the pinned leaderboard commit gives 4.35 macro WER (8.00 / 5.89 / 7.18 / 1.17 / 2.76 / 2.70 / 2.72). Please run the scripted and conversational private sets when capacity permits. Thanks!

**Comment by warlockee** [comment]
Update: the previous 4.35 revision in my earlier comment has been superseded. Please evaluate exact model revision c25965e1d8a449b274cf496b185f8216db1296c7. Its complete official public scores are 7.98 / 5.77 / 7.17 / 1.19 / 2.75 / 2.70 / 2.74, mean 4.33. The PR body and model evaluation metadata are updated accordingly. The private scripted and conversational suite is still requested.

**Comment by warlockee** [comment]
Final promotion update: please evaluate immutable model revision bce834cee3d106a9f0ec1ba98e96d995f13f59de. Its complete official public scores are 7.98 / 5.78 / 7.18 / 1.19 / 2.75 / 2.70 / 2.73, mean 4.33, at beam size 4 and length penalty 0.7. The model card and .eval_results metadata now match this exact revision. The private scripted and conversational suite is still requested.

File: transformers/submit_jobs_qwen3asr.sh
Comment (ebezzam) [inline transformers/submit_jobs_qwen3asr.sh:92]: can you edit to following to pass a revision tag
```
      --max_new_tokens=${MAX_NEW_TOKENS} \
      ${REVISION_ARG} &&
```

File: transformers/submit_jobs_qwen3asr.sh [hunk 1]
```diff
@@ -35,6 +35,7 @@ fi
 MODEL_IDs=(
     "Qwen/Qwen3-ASR-0.6B-hf"
     "Qwen/Qwen3-ASR-1.7B-hf"
+    "bosonai/Qwen3-ASR-1.7B-hf-orze"
```
Comment (ebezzam) [inline transformers/submit_jobs_qwen3asr.sh:38]:
```suggestion
    "bosonai/Qwen3-ASR-1.7B-hf-orze bce834cee3d106a9f0ec1ba98e96d995f13f59de"
```

File: transformers/submit_jobs_qwen3asr.sh
Comment (ebezzam) [inline transformers/submit_jobs_qwen3asr.sh:59]: edit to something like below to pass revision tag
```
for model_cfg in "${MODEL_IDs[@]}"; do
    read -r MODEL_ID REVISION <<< "$model_cfg"
    REVISION_ARG=""
    [[ -n "$REVISION" ]] && REVISION_ARG="--revision=${REVISION}"
    MODEL_FOLDER="${MODEL_ID//\//-}"

    echo "████████████████████████████████████████████████████████████████████████████████"
    echo "  Evaluating: ${MODEL_ID}${REVISION:+ (revision=${REVISION})}"
    echo "████████████████████████████████████████████████████████████████████████████████"

```

**Review by ebezzam — COMMENTED** [review]
thanks @warlockee for the PR! I've launched the eval. Can you make the suggested changes to be able to pass the revision?

File: transformers/submit_jobs_qwen3asr.sh (hunk 1 above)
Comment (ebezzam) [reply transformers/submit_jobs_qwen3asr.sh:38]: we should also specify batch size for each model like here, as it seems like your checkpoint needs a smaller one? (and keep 256 for the others)

see [here](https://github.com/huggingface/open_asr_leaderboard/blob/e7cb8e9b79ab908302c30bdc9f38152ad666a43c/transformers/submit_jobs_whisper.sh#L36) for example

**Comment by ebezzam** [comment]
@warlockee I obtain these results if you can confirm

```
********************************************************************************
Results per dataset:
********************************************************************************
bosonai/Qwen3-ASR-1.7B-hf-orze | ArtificialAnalysis-Earnings22-Cleaned-AA-chunked__test: WER = 5.85 %, RTFx = 64.65
bosonai/Qwen3-ASR-1.7B-hf-orze | VoiceArena-Monsoon_en_IN_test__test: WER = 3.50 %, RTFx = 163.58
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_ami_cleaned_test: WER = 8.00 %, RTFx = 115.24
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_gigaspeech_cleaned_test: WER = 7.18 %, RTFx = 194.42
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_librispeech_test.clean: WER = 1.19 %, RTFx = 209.35
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_librispeech_test.other: WER = 2.76 %, RTFx = 200.10
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_spgispeech_test: WER = 2.71 %, RTFx = 271.17
bosonai/Qwen3-ASR-1.7B-hf-orze | hf-audio-open-asr-leaderboard_voxpopuli_cleaned_aa_test: WER = 2.73 %, RTFx = 182.82

********************************************************************************
Composite Results:
********************************************************************************
bosonai/Qwen3-ASR-1.7B-hf-orze: WER = 4.24 %
bosonai/Qwen3-ASR-1.7B-hf-orze: RTFx = 218.98
********************************************************************************
```
